pkg/manager/member/tidb_member_manager.go

```diff
@@ -1049,14 +1049,14 @@ func (m *tidbMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	for id := range helper.GetPodOrdinals(tc.Status.TiDB.StatefulSet.Replicas, set) {
 		name := fmt.Sprintf("%s-%d", controller.TiDBMemberName(tc.GetName()), id)
 		health, err := m.deps.TiDBControl.GetHealth(tc, int32(id))
-		if err != nil {
-			return err
-		}
-
 		newTidbMember := v1alpha1.TiDBMember{
 			Name:   name,
 			Health: health,
 		}
+		if err != nil {
```

Comment:
do we need to keep the `err` in `_, err = getBodyOK(httpClient, url)`?

Reply:
why do you revert it in https://github.com/pingcap/tidb-operator/pull/4844/commits/a637f81cfe5e78504bfa02a01a5ed38c1c6a5566